lldb/source/Plugins/Process/gdb-remote/ProcessGDBRemote.cpp

```diff
@@ -2611,7 +2620,11 @@ StateType ProcessGDBRemote::SetThreadStopInfo(StringExtractor &stop_packet) {
         StringExtractorGDBRemote extractor(value);
         if (std::optional<GPUActions> gpu_actions = 
             extractor.GetFromJSONHexASCII<GPUActions>()) {
-          HandleGPUActions(*gpu_actions);
+          if (Status err = HandleGPUActions(*gpu_actions); err.Fail()) {
```

Comment:
ditto

Reply:
initializer syntax should be fine on MSCV. That has been around for many years already.
And this pattern is actually used in a few places in LLDB and LLVM already, so it should be fine.